benchexec/tools/graves-par.py

```diff
@@ -0,0 +1,78 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.coveriteam as coveriteam
+import benchexec.result as result
+from benchexec.tools.template import UnsupportedFeatureException
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import re
+
+class Tool(coveriteam.Tool):
```

Comment:
This is somewhat confusing. Currently, the class inherits from `coveriteam.Tool`, but it overwrites all methods from `coveriteam.Tool`, so this is useless. On the other hand, it contains two methods copied verbatim from `coveriteam.Tool` (`version` and `program_files`) and one copied verbatim from `coveriteam-verifier-validator`.

The main question here is: What is the relationship between CoVeriTeam and Graves-Par?
Is Graves-Par a variant of CoVeriTeam that will follow CoVeriTeam's development, or is it a tool that will be developed independently.
Suppose for example that for a new version of CoVeriTeam we make adjustments to its tool-info module, should the tool-info module of Graves-Par also get these adjustments automatically, or not?
And does the answer of this question apply to the base CoVeriTeam or to the specific coveriteam-verifier-validator instance?

Based on this it should be decided whether to use inheritance or not. If inheritance is preferable, then the duplicate code should be removed.

Reply:
Graves-Par is a variant of coveriteam. We want to use their infrastructure and allow our selector to configure coveriteam. Inheritance would be preferred. I will make the appropriate changes.